### PR TITLE
Upgrade to v1.32.0 of the Azurerm Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

**Comment by mikhailshilkov** [comment]
@stack72 I noticed some fresh deprecation warnings so I adjusted the usage to remove those. I can't make sense of the errors though...

File: examples/examples_test.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ func TestExamples(t *testing.T) {
 		base.With(integration.ProgramTestOptions{Dir: path.Join(cwd, "http-multi")}),
 		base.With(integration.ProgramTestOptions{Dir: path.Join(cwd, "iot")}),
 		base.With(integration.ProgramTestOptions{Dir: path.Join(cwd, "queue")}),
-		base.With(integration.ProgramTestOptions{Dir: path.Join(cwd, "table")}),
+		//base.With(integration.ProgramTestOptions{Dir: path.Join(cwd, "table")}),
```
Comment (jen20) [inline examples/examples_test.go]: Why does this one need disabling?

**Comment by lukehoban** [comment]
@stack72 Are we going to skip this and jump straight to 0.32.1 now?

**Comment by stack72** [comment]
Closing this PR - v1.32.0 had regressions around storage - a v1.32.1 release is imminent

**Pull request closed without merging**